geocube/geo_utils/geobox.py

```diff
@@ -193,12 +187,12 @@ def from_vector(self, vector_data):
             raise RuntimeError("Must specify 'resolution' if 'like' not specified.")
 
         if self.output_crs:
-            crs = geometry.CRS(crs_to_wkt(self.output_crs))
+            crs = CRS(crs_to_wkt(self.output_crs))
         else:
-            crs = geometry.CRS(crs_to_wkt(vector_data.crs))
+            crs = CRS(crs_to_wkt(vector_data.crs))
 
```

Comment:
also all places that accept crs parameter will perform auto-conversion to `odc.geo.crc.CRS`, so don't have to convert in your code for the most part.